lib/ember-cli/asset_resolver.rb

```diff
@@ -1,8 +1,8 @@
 module EmberCLI
   class AssetResolver
-    def initialize(options)
-      @app = options.fetch(:app)
-      @sprockets = options.fetch(:sprockets)
+    def initialize(app:, sprockets:)
```

Comment:
@seanpdoyle Can you explain this syntax?  I've never seen arguments with colons like that